docs/tracing_ui.md

```diff
@@ -118,7 +118,7 @@ All operations and information regarding trees are organized under a tab called
 A typical skeleton annotation consists of one or more trees.
 Trees can be nested and organized in so-called `Tree Groups`.
 Tree groups can have a name and are used to structure and label your annotation even further.
-Trees can be dragged and dropped between tree groups. This action can be applied to multiple trees by selecting them with Ctrl + Left Mouse. 
+Trees can be dragged and dropped between tree groups. This action can be applied to multiple trees by selecting them with Ctrl + Left Mouse. On Mac OS systems the keys for the multiple selection are Command + Left Mouse.
```

Comment:
:+1:
I'd put it a bit more concise `... selecting them with Ctrl + Left Mouse (on Mac OS Cmd + Left Mouse).`